build/build_release.sh

```diff
@@ -294,8 +294,7 @@ else
    echo "Skipping mvn clean install -P communityRelease -DskipTests"
 fi
 
-
-mvn assembly:attached $MAVEN_FLAGS
+mvn -f src/pom.xml assembly:single $MAVEN_FLAGS -N
```

Comment:
The `pushd src` a few lines above means you don't need `-f src/pom.xml` (more specifically, as written the build will fail - remove `-f src/pom.xml` )